apps/docs/src/getting-started.md

```diff
@@ -62,6 +67,28 @@ function App() {
 export default App;
 ```
 
+## Browser
+
+For a quick test or just playing around, you can load it in your Web Apps straight from our CDN.
```

Comment:
Is it worth noting here that this is only supported from `0.73.0`?

Reply:
I thought the same, but the version is self-contained in the snippet URL.


![version](https://github.com/FuelLabs/fuels-ts/assets/26660/87402959-c4d0-4a71-91a1-e43750fd4cb6)